### DISCUSSION TITLE
PersistentTokenCache Sample - File Created/Modified update issues

### DISCUSSION THREAD

**Discussion by Sniperumm** [pnp/pnpcore#1082] [Q&A] [answered]
Hi,

I'm attempting to use the [Demo.PersistentTokenCache sample](https://pnp.github.io/pnpcore/demos/Demo.PersistentTokenCache/README.html) to update both the Created and Modified properties of a single file.  For some reason both Created and Modified don't get updated when UpdateOverwriteVersionAsync is ran. Instead, it just updates the Modified property with the time the update is executed. If I run this code with the InteractiveAuthenticationProvider it works correctly. 

Scope's set:

Sites.Manage.All (Graph)
User.Read (Graph)
AllSites.Manage (SharePoint)

Here's my file update code:

```csharp
var doc = await context.Web.Lists.GetByTitleAsync("Documents", p => p.Fields);

var file = doc.RootFolder.Files.Where(x => x.Name == "testing123.txt").FirstOrDefault();

await file.LoadAsync(p => p.ListItemAllFields);

var newDate = new DateTime(2020, 10, 20);

file.ListItemAllFields["Created"] = newDate;
file.ListItemAllFields["Modified"] = newDate;

await file.ListItemAllFields.UpdateOverwriteVersionAsync();
```

Any idea's why this might be happening?

**Comment by Sniperumm** [comment, marked as answer]
So not long after posting this I've got it working. It required me to change the SharePoint AllSites.Manage (Delegated) scope to SharePoint AllSites.FullControl (Delegated). Little confused on why AllSites.Manage doesn't work for PersistentTokenCache but it works fine with InteractiveAuthenticationProvider?

**Reply by jansenbe** [reply]
@Sniperumm : when you want to override the `Created`, `Modified`, `Author` and `Editor` property you need "full control". See https://pnp.github.io/pnpcore/using-the-sdk/listitems-intro.html#updating-the-list-item-author-editor-created-and-modified-system-properties for more details.

**Reply by Sniperumm** [reply]
Thank you Jansenbe :)
